### PR TITLE
Silent fail

### PR DESCRIPTION
**Pull request by angelbarrera92**
Ciao!

From @ralgozino's feedback, I've done a simple modification to the CLI. In case of error, the usage help is not shown anymore. Thanks!

### Before

```bash
$ docker run -it --rm registry.sighup.io/poc/tls-certs-check:v0.1.0 ./tls-certs-check --host expired.badssl.com 
ERRO[0000] server doesn't support SSL                    addr="expired.badssl.com:443" error="x509: certificate has expired or is not yet valid: current time 2021-09-13T15:45:46Z is after 2015-04-12T23:59:59Z"
Error: x509: certificate has expired or is not yet valid: current time 2021-09-13T15:45:46Z is after 2015-04-12T23:59:59Z
Usage:
  tls-certs-check [flags]

Flags:
      --days int           number of days before the certificate expires to throw an error (default 7)
  -h, --help               help for tls-certs-check
      --host string        host to check the SSL certificate
      --log-level string   logging level (debug, info...) (default "info")
      --port int           port to check the SSL certificate (default 443)

FATA[0000] error in the cli. Exiting                     error="x509: certificate has expired or is not yet valid: current time 2021-09-13T15:45:46Z is after 2015-04-12T23:59:59Z"
```

### After

```bash
$ docker run -it --rm docker.io/library/tls-certs-check:local-build ./tls-certs-check --host expired.badssl.com 
ERRO[0000] server doesn't support SSL                    addr="expired.badssl.com:443" error="x509: certificate has expired or is not yet valid: current time 2021-09-13T15:44:11Z is after 2015-04-12T23:59:59Z"
Error: x509: certificate has expired or is not yet valid: current time 2021-09-13T15:44:11Z is after 2015-04-12T23:59:59Z
```

### CODE REVIEW

File: cmd/tls-certs-check/main.go [hunk 1]
```diff
@@ -52,7 +53,6 @@ func init() {
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.WithError(err).Fatal("error in the cli. Exiting")
```
Comment (ralgozino) [inline cmd/tls-certs-check/main.go:55]: Are we sure that removing this we don't fail silently when we shouldn't? if I miss-type something in the command for example

Comment (angelbarrera92) [reply cmd/tls-certs-check/main.go:55]: What does silent error is the `SilenceUsage` configuration parameter of the `rootCmd` struct.
This `log.Fatal` sentence is just a redundant error message.

Comment (ralgozino) [reply cmd/tls-certs-check/main.go:55]: excellent! thanks

**Review by ralgozino — APPROVED** [review]

File: cmd/tls-certs-check/main.go (hunk 1 above)
Comment (nandajavarma) [reply cmd/tls-certs-check/main.go:55]: @angelbarrera92 it is not quite clear to me how this is a redundant message. for example: https://github.com/sighupio/tls-certs-check/blob/095b60ffdb92fe6825b4537fbe01ba0f232414e1/pkg/tls-certs-check/main.go#L63 seems to return an error which is simply returned by `TLSCheck` function. shouldn't we be logging in out somewhere?

Comment (angelbarrera92) [reply cmd/tls-certs-check/main.go:55]: The `checkCertificateDate` function returns an error only if the certificate is going to expire in a time window defined from the `days` function argument.
The `TLSCheck` just checks if the certificate is experired.

**Review by nandajavarma — APPROVED** [review]

**Review by emasacco — APPROVED** [review]

**Review by ipedrazas — APPROVED** [review]
lgtm

**Pull request merged**